gitlab/v4/objects.py

```diff
@@ -2739,6 +2772,7 @@ class Project(SaveMixin, ObjectDeleteMixin, RESTObject):
     _short_print_attr = 'path'
     _managers = (
         ('accessrequests', 'ProjectAccessRequestManager'),
+        ('approvals', 'ProjectMergeRequestApprovalManager'),
```

Comment:
The classes you've defined are attached link to merge requests, so this change should be move to the `ProjectMergeRequest` class I think.

Reply:
Since the endpoint I implemented is `/projects/%(project_id)s/approvals` and not `/projects/%(project_id)s/merge_requests/%(merge_request_iid)s/approvals`, I feel this is adequate.

Maybe the object name is off ? I could rename the manager `ProjectMergeRequestApprovalConfigurationManager` if that would make more sense. I tried to keep the same semantics as the GitLab API doc for consistency, but clarity is obviously desirable.

Reply:
I renamed to `('approvalsettings', 'ProjectMergeRequestApprovalSettingsManager')`. Hopefully this makes the purpose of this object clearer.